Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/messaging/service/decorator/HttpHeaderServiceEndpointDecoratorTest.java

```diff
@@ -101,6 +101,40 @@ protected PropertyAccessor getPropertyAccessor() {
     }
     
     @Test
+    public void testConfigure_WithoutValues() throws PropertyAccessException {
+        CONNECTTestClient<TestServicePortType> testClient = new CONNECTTestClient<>(
+                new TestServicePortDescriptor());
+
+        ServiceEndpoint<TestServicePortType> serviceEndpoint = testClient.getServiceEndpoint();
+        
+        final PropertyAccessor mockPropAccessor = mock(PropertyAccessor.class);
+
+        AssertionType assertion = new AssertionType();
+        assertion.setKeepAlive("true");
+        
+        CONNECTCustomHttpHeadersType assertionHeaders = new CONNECTCustomHttpHeadersType();
+        assertionHeaders.setHeaderName("headerName");
+        assertionHeaders.setHeaderValue("headerValue");
+        
+        assertion.getCONNECTCustomHttpHeaders().add(assertionHeaders);
+
+       HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(serviceEndpoint, assertion) {
```

Comment:
In place of HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(serviceEndpoint, assertion) {}; use the below statement since you are not overriding any methods:
HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(serviceEndpoint, assertion);